.github/workflows/dev-docker-apps.yml

```diff
@@ -32,22 +32,44 @@ jobs:
         id: git_vars
         run: echo ::set-output name=COMMIT_HASH::$(git rev-parse --short HEAD)
 
-      - name: "Login to DockerHub"
-        uses: docker/login-action@v1
+      - name: "Configure Docker"
+        run: echo ${{ secrets.CR_PAT }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
+
+      - name: "Set up Buildx"
+        id: buildx
+        uses: docker/setup-buildx-action@v1
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          version: latest
 
-      - name: "Docker build"
-        run: docker build -f ./Dockerfile-${{ matrix.app }} -t $REPO_OWNER/zeth-${{ matrix.app }}:git-${{ steps.git_vars.outputs.COMMIT_HASH }} .
+      - name: "Build and push image git-SHA"
+        run: |
+          docker buildx build \
+            --tag ghcr.io/$GITHUB_REPOSITORY:git-${{ steps.git_vars.outputs.COMMIT_HASH }}-${{ matrix.app }} \
+            --cache-from=type=registry,ref=ghcr.io/$GITHUB_REPOSITORY:cache-${{ matrix.app }} \
+            --cache-to=type=registry,ref=ghcr.io/$GITHUB_REPOSITORY:cache-${{ matrix.app }} \
+            --platform linux/amd64 \
+            --output "type=image,push=true" \
+            --file ./Dockerfile-${{ matrix.app }} ./
 
-      - name: "Docker push"
-        run: docker push $REPO_OWNER/zeth-${{ matrix.app }}:git-${{ steps.git_vars.outputs.COMMIT_HASH }}
+  test:
+    name: "Test"
+    needs: docker-build
+    runs-on: ubuntu-20.04
+    timeout-minutes: 360
 
-      - name: "Docker Info"
-        run: |
-          docker inspect $REPO_OWNER/zeth-${{ matrix.app }}:git-${{ steps.git_vars.outputs.COMMIT_HASH }}
-          docker history $REPO_OWNER/zeth-${{ matrix.app }}:git-${{ steps.git_vars.outputs.COMMIT_HASH }} --no-trunc
-          docker version
-          docker info
-          uname -a
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Get vars from git"
+        id: git_vars
+        run: echo ::set-output name=COMMIT_HASH::$(git rev-parse --short HEAD)
+
+      - name: "Configure Docker"
+        run: echo ${{ secrets.CR_PAT }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
+
+      - name: "Run service"
+        run: echo "Here we will run docker-compose based on images git-"${{ steps.git_vars.outputs.COMMIT_HASH }}
+
+      - name: "Test RPC"
+        run: echo "Here we will run a tests"
```

Comment:
Not sure I get the purpose of this job @eastata 
What are the "tests" doing here, beyond the `echo` messages. You don't seem to do a `docker-compose up` (as suggested in the `Run service` step above, do you?)

Reply:
Yes. It is placeholder for docker-compose.
I guess the integral tests should be implemented at another issue/PR

Reply:
I see, thanks @eastata 
Let's comment this job out for now. I think it is not necessary to run an extra job if it only aims at being a placeholder and run `echo`s. We'll see if we need to do extra tests with `docker-compose`, and add them in a later PR if necessary (bearing in mind that @dtebbs added a few test jobs in #326 already)

Reply:
Removed